lib/ferrum/page.rb

```diff
@@ -262,7 +262,7 @@ def prepare_page
       end
 
       if @browser.options[:save_path]
-        command("Page.setDownloadBehavior", behavior: "allow", downloadPath: @browser.options[:save_path])
+        command("Browser.setDownloadBehavior", behavior: "allow", downloadPath: @browser.options[:save_path], eventsEnabled: true)
```

Comment:
regarding `emits additional Browser.downloadProgress event`, how about set it as option?
Like: `options[:download_events_enabled]`
I also feel like a good to rename `save_path` to `download_path` or even better to have `download_behavior` option as a hash with `path` and `events_enabled` option. However it would be a breaking change, so we can apply those changes for next release.

Reply:
ETooManyOptions ;) good for starters I think

Reply:
`save_path` is also where screenshots, PDF, and MHTML files are saved, sort of. If you're running with the `url` option then screenshots are saved at `save_path` but other downloads are not. I'm not sure about PDF or MHTML but if you for instance fetch a .zip file from somewhere it'll be saved wherever the `url` chrome browser is running.

Reply:
Though previosly it was not needed for more options coz [Page.setDownloadBehavior](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-setDownloadBehavior) has only `downloadPath` as optional option, case of [Browser.setDownloadBehavior](https://chromedevtools.github.io/devtools-protocol/tot/Browser/#method-setDownloadBehavior) required such more options, coz  like `browserContextId` and `eventsEnabled`.
> fetch a .zip file from somewhere it'll be saved wherever the url chrome browser is running

I think it's default behavior, so the path of storage should be manageable by chore options as well.

Reply:
Oh I missed `browserContextId` in a rush yesterday. Seems like it's the missing piece to make it work without `Page.setDownloadBehavior` on my mac, turns out it's not very optional as the doc says. I'll update the code